src/pages/dashboard/tables/Tables.tsx

```diff
@@ -7,11 +7,11 @@ import {getTableData} from '../../../global/dataParsing'
 import {action} from '../../../global/util'
 import {colors} from '../graphs/Graphs'
 import {CustomTable} from './CustomTable'
-import {ColumnEntry, Overview, TableOverview} from '../../../global/typeUtils'
+import {ColumnEntry, Overview} from '../../../global/typeUtils'
 
 interface TableComponentProps extends WithTranslation {
   overview: Overview;
-  tableOverview: TableOverview;
+  tableOverview: {};
```

Comment:
There is no specific type for tthe tableOverview?

Reply:
There was, but it didn't play nicely with the `TABLE_TYPE` because it was a const instead of an enum. I could not think of a better structure at the time.
![Screenshot 2020-07-14 at 10 09 03](https://user-images.githubusercontent.com/3997576/87401673-b9cc8a00-c5ba-11ea-8d33-b4614b587bae.png)

After looking a bit more over it, I think I came up with a somewhat clean solution by creating another enum, so I did a commit now.

Reply:
Nice 👍